docs/root/intro/version_history.rst

```diff
@@ -11,6 +11,9 @@ Version history
 * health check: added support for :ref:`custom health check <envoy_api_field_core.HealthCheck.custom_health_check>`.
 * health_check: added support for :ref:`health check event logging <arch_overview_health_check_logging>`.
 * http: better handling of HEAD requests. Now sending transfer-encoding: chunked rather than content-length: 0.
+* http: no longer adding whitespace when appending X-Forwarded-For headers. **Warning**: this is not
+  compatible with 1.7.0 builds prior to `9d3a4eb4ac44be9f0651fcc7f87ad98c538b01ee <https://github.com/envoyproxy/envoy/pull/3610>`_.
+  See `#3611 <https://github.com/envoyproxy/envoy/issues/3611>`_ for details.
```

Comment:
Hm, on second thought, I wonder if we should wait one more release cycle on this for good form.  If everyone else is running from HEAD or close to it, I think we're fine.  If anyone is running on release builds, it'll make rolling from 1.7 to 1.8 correctly a bit tricky which arguably it shouldn't be.

Reply:
I think I agree (although we run from HEAD), sounds like two release should avoid surprises....